docs/src/api/ulti-tools-plugin.md

```
@@ -37,7 +37,8 @@ public abstract class **UltiToolsPlugin** extends java.lang.Object implements IP
 
 | 限定符和类型                                          | 方法和说明                                                                       |
 |-------------------------------------------------|-----------------------------------------------------------------------------|
-| `static CommandManager `                        | static CommandManager                                                       |
+| `static CommandManager `                        | getCommandManager\()                                                        |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)